.env

```diff
@@ -0,0 +1,17 @@
+APP_NAME=readerapi
+
+NODE_ENV=development
+HOST=localhost
+PORT=3000
+
+DB_HOST=localhost
+DB_PORT=27017
+DB_NAME=reader-api-dev
+DB_DEBUG=false
+
+SESSION_SECRET=readerapi
+
+
+FACEBOOK_CLIENT_ID=1922383334663210
+FACEBOOK_CLIENT_SECRET=23d21a8e55c42af2fe765153490a8e16
+FACEBOOK_CALLBACK=http://localhost:3000/api/v1/auth/facebook/callback
```

Comment:
Remove all values pls. I said empty .env file. This will cause credential data leaked.